File: Programming_Task_04_ShaliniSengar/Python_Files/Marks_Analysis_System.py
marks = []
total = 0

print("Enter marks for 5 subjects:")

for i in range(5):
    mark = int(input(f"Subject {i+1}: "))
    marks.append(mark)
    total += mark

percentage = total / 5

if percentage >= 90:
    grade = "A"
elif percentage >= 80:
    grade = "B"
elif percentage >= 70:
    grade = "C"
elif percentage >= 60:
    grade = "D"
else:
    grade = "F"

print("\n----- Result -----")
print("Total Marks =", total)
print("Percentage =", round(percentage, 2), "%")
print("Grade =", grade)

File: Programming_Task_04_ShaliniSengar/C_Files/Marks_Analysis_System.c
#include <stdio.h>

int main() {
    int marks[5];
    int total = 0;
    float percentage;
    char grade;

    printf("Enter marks for 5 subjects:\n");

    for (int i = 0; i < 5; i++) {
        printf("Subject %d: ", i + 1);
        scanf("%d", &marks[i]);
        total += marks[i];
    }

    percentage = total / 5.0;

    if (percentage >= 90)
        grade = 'A';
    else if (percentage >= 80)
        grade = 'B';
    else if (percentage >= 70)
        grade = 'C';
    else if (percentage >= 60)
        grade = 'D';
    else
        grade = 'F';

    printf("\n----- Result -----\n");
    printf("Total Marks = %d\n", total);
    printf("Percentage = %.2f%%\n", percentage);
    printf("Grade = %c\n", grade);

    return 0;
}